common/lib/xmodule/xmodule/js/fixtures/problem_content.html

```diff
@@ -16,6 +16,10 @@ <h2 class="problem-header">Problem Header</h2>
     <button class="reset">Reset</button>
     <button class="save">Save</button>
     <button class="show"><span class="show-label">Show Answer(s)</span> <span class="sr">(for question(s) above - adjacent to each field)</span></button>
+    <div class="reset-error">
```

Comment:
@mehrozezahid use [is-hidden](https://github.com/edx/edx-platform/blob/master/common/static/sass/_mixins.scss#L376) class to hide an element. And when you want to show the element remove the `is-hidden` class